### PR TITLE
chore(ApiRequest): remove mention of var

### PR DESCRIPTION
**Pull request by alanlichen**
**Please describe the changes this PR makes and why it should be merged:**

Remove the usage of var in APIRequest class

**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

File: src/rest/APIRequest.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ const AbortController = require('abort-controller');
 const fetch = require('node-fetch');
 const { UserAgent } = require('../util/Constants');
 
-if (https.Agent) var agent = new https.Agent({ keepAlive: true });
+let agent;
+if (https.Agent) agent = new https.Agent({ keepAlive: true });
```
Comment (vladfrangu) [inline src/rest/APIRequest.js:10]: Might as well make this a constant, since d.js runs only on node now

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: src/rest/APIRequest.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const AbortController = require('abort-controller');
 const fetch = require('node-fetch');
 const { UserAgent } = require('../util/Constants');
 
-if (https.Agent) var agent = new https.Agent({ keepAlive: true });
+const agent = new https.Agent({ keepAlive: true });
```
Comment (DTrombett) [inline src/rest/APIRequest.js:9]: Is this always defined?
```suggestion
const agent = new https.Agent?.({ keepAlive: true });
```

Comment (alanlichen) [reply src/rest/APIRequest.js:9]: It should always be defined 🤔

Comment (DTrombett) [reply src/rest/APIRequest.js:9]: Ohh lol, didn't notice Vlad's comment. So sorry :+1:

**Review by DTrombett — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Review by joseywoermann — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Pull request merged**